dynamic_programming/matrix_chain_order.c

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file
+ * @brief [Matrix Chain Order] (https://en.wikipedia.org/wiki/Matrix_chain_multiplication)
```

Comment:
This will make it a valid Markdown link.
```suggestion
 * @brief [Matrix Chain Order](https://en.wikipedia.org/wiki/Matrix_chain_multiplication)
```